examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,82 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the regularization II
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter.
+
+In this example the dependent variable Y is set as a function
+of the input features: Y = Xa + bias. The coefficient vector a is
+randomly sampled from a normal distribution, as well as the bias
+term.
```

Comment:
Could you write this as `Y = X*w + c`. And note that the bias is no longer randomly sampled. Also you may add that this is non-noisy data, so it will be possible to extract the exact ground-truth of the coefficients.